subsystem/subsystem-core/pom.xml

```diff
@@ -111,6 +111,11 @@
             <version>1.2.101.v20150831-1342</version>
         </dependency>
         <dependency>
+            <groupId>com.google.guava</groupId>
+            <artifactId>guava</artifactId>
```

Comment:
Please check also file https://github.com/apache/aries/blob/trunk/subsystem/readme.txt 

IMHO guava should be added here.